pkgs/development/interpreters/lua-5/interpreter.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, readline
 , compat ? false
 , callPackage
-, makeWrapper
+, makeBinaryWrapper
```

Comment:
Please keep the input like it is and use an override to easily switch between the wrappers which are interchangeably.

Reply:
> Please keep the input like it is and use an override to easily switch between the wrappers which are interchangeably.

I'm not sure we want to imply that changing back the `makeWrapper` implementation to the previous one is guaranteed to work. Changing the implementation here implies that we wish to support only this implementation and that we don't want to maintain compatibility with the old implementation.

Reply:
The override would need to be done here https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/lua-5/default.nix

Reply:
where should we find libtexpdf ? `nix-locate libtexpdf` returned no result for me, is it just because it's unpackaged ? 
As for the binary wrapper, seems like without it lua wrapping doesn't work on darwin in which case we could enable it everywhere as sandro advises. Alternatively if you want a quick merge, we can just do:
```
sile = callPackage ../tools/typesetting/sile {
    lua = lua5_4;
    makeWrapper = makeBinaryWrapper;
  };
 ```

Reply:
Just for reference: On regular Linux systems it is installed to `/usr/lib/libtexpdf.so` when following the usual autotools `./configure --prefix=/usr && make install`.